From 91fccf5cc559543122c76694f4e39befae191853 Mon Sep 17 00:00:00 2001
From: Andrew Robbins <contact@andrewrobbins.info>
Date: Sat, 4 Jul 2020 12:26:18 -0500
Subject: [PATCH] Enable serial console

Necessary for boards such as KCMA-D8 or KGPE-D16 where
the coreboot framebuffer is not supported and/or a headless
setup is desired.
---
 config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.h b/config.h
index 9f7c7a5..36eaa9d 100644
--- a/config.h
+++ b/config.h
@@ -38,7 +38,7 @@
 /*	This is normally off since it slows down testing.  Change to a 1 */
 /*	to enable. */
 #ifndef SERIAL_CONSOLE_DEFAULT
-#define SERIAL_CONSOLE_DEFAULT 0
+#define SERIAL_CONSOLE_DEFAULT 1
 #endif
 
 /* SERIAL_TTY - The default serial port to use. 0=ttyS0, 1=ttyS1, 2=ttyS2, 3=ttyS3 */
-- 
2.27.0

